index.html

```diff
@@ -0,0 +1,162 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WEB3 HACKATHON</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="./assets/favicons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./assets/favicons/favicon-16x16.png">
+    <link rel="manifest" href="./assets/favicons/site.webmanifest">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,600,0,0" />
+</head>
+<body>
+    <section  id="hero">
+        <ul class="top-menu">
+            <li><a href="" class="top-nav-link"><img src="https://img.icons8.com/small/32/FFFFFF/facebook.png"/></a></li>
+            <li><a href="" class="top-nav-link"><img src="https://img.icons8.com/small/32/FFFFFF/twitter.png"/></a></li>
+            <li><a href="" class="top-nav-link">English</a></li>
+            <li><a href="" class="top-nav-link">My Page</a></li>
+            <li><a href="" class="top-nav-link">Logout</a></li>
+        </ul>
+        <div class="hero-section">
+        <nav class="nav-bar">
+                <a href="/" class="logo">
+                    <img src="./assets/images/Web3-Hackathon.png" alt="web3 Hackathon Logo">
+                </a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            <ul class="menu">
+                <li><a href="./about.html" class="nav-link">About</a></li>
+                <li><a href="./#program" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li><a href="" class="nav-link">CC Campaign</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] for better semantics, it's better to wrap this section in `header` tag instead of `section` 👍